source/components/Autocomplete.js

```diff
@@ -67,12 +68,10 @@ class Autocomplete extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
 
-    // $FlowFixMe
-    this.rootElement = React.createRef();
-    // $FlowFixMe
-    this.inputElement = React.createRef();
-    // $FlowFixMe
-    this.suggestionsElement = React.createRef();
+    // define refs
+    this.rootElement = createRef();
```

Comment:
Both the createRef and forwardRef methods from create-react-ref will check if the methods exist on the React module before using the polyfill.